src/actions/ActionCreator.js

```diff
@@ -148,4 +158,3 @@ export function enterDebug() {
     dispatch({ type: actionTypes.DEBUG_MODE, payload: '' });
```

Comment:
@minkeesung just a note, we shouldn't be running debug mode in production. We r going to performance issue on the user end.